src/datumaro/plugins/validators.py

```diff
@@ -547,11 +547,24 @@ def compute_statistics(self, dataset):
 
         stats, filtered_anns = self._compute_common_statistics(dataset)
 
+        label_cat = dataset.categories()[AnnotationType.label]
+        dm_label_groups = label_cat.label_groups
+
+        label_name_to_group = {}
+        for dm_label_group in dm_label_groups:
+            for label_name in dm_label_group.labels:
+                label_name_to_group[label_name] = dm_label_group.name
+
         stats["items_with_multiple_labels"] = []
         for item_key, anns in filtered_anns:
-            ann_count = len(anns)
-            if ann_count > 1:
-                stats["items_with_multiple_labels"].append(item_key)
+            occupied_groups = set()
+            for ann in anns:
+                label_name = label_cat.items[ann.label].name
+                label_group = label_name_to_group.get(label_name, "")
```

Comment:
This will invoke L565 although `dm_label_groups` is an empty list since `label_group` will be `""` in that situation.

In addition, we might need to consider `group_type`. I feel that this logic should be effective only for `group_type == "exclusive"`
https://github.com/openvinotoolkit/datumaro/blob/04dff62070c4e51683e6133c88f0187e479b91b1/src/datumaro/components/annotation.py#L122-L126

Lastly, `group_type` seems selectable one, we can make it clearer by changing it to `Enum`.

Reply:
Please see c2f6cc06f25d787b488970737e3732db5212ebf8.